### PR TITLE
fix modal scrolling performance on mobile

### PR DESCRIPTION
**Pull request by tim-smart**
This is primarily done by removing the blur effect from the overlay, and
replacing it will a translucent black.

The blur effect performs poorly on mobile devices.

### CODE REVIEW

**Comment by youngcw** [comment]
Can this be more pronounced?  Currently its hard to even notice that there is a change at all.

**Comment by tim-smart** [comment]
> Can this be more pronounced?  Currently its hard to even notice that there is a change at all.

The goal was to make only a small visual difference, but improve the scrolling speed of the category selector menu on mobile.

**Comment by joel-jeremy** [comment]
Are you using Safari by any chance? I remember implementing something on the PrivacyFilter to fix slow blur performance:
image

**Comment by tim-smart** [comment]
Android / chrome. I tried the forcing gpu rendering trick, but it didn't have any effect. The difference here is that it was using `backdrop-filter`.

**Comment by tim-smart** [comment]
Here is some videos showing the difference:

https://github.com/user-attachments/assets/b9facb34-c411-4bb1-af69-4d0250488379
https://github.com/user-attachments/assets/302af698-019c-467a-a8d1-5446ed066b5b

**Review by MikesGlitch — COMMENTED** [review]
I can confirm on Mobile Chrome and Edge there's a slight performance improvement. 

We lose the blur though and there's not a lot of contrast with some other elements on the screen in desktop mode. 

E.g. 
image

What are peoples thoughts on if the style was only applied to mobile?

**Comment by tim-smart** [comment]
I think mobile only is a good option.

File: packages/desktop-client/src/components/common/Modal.tsx
Comment (tim-smart) [reply packages/desktop-client/src/components/common/Modal.tsx:86]: It forces the browser to render the layer with gpu acceleration.

**Review by MikesGlitch — COMMENTED** [review]
Gotta say I'm really thinking the blur is doing good work on these screens:

image

Are we sure there's no way to keep the blur but improve the performance? I'm finding it difficult to decide if it's worth losing the blur for the tiny perf improvement.

**Comment by joel-jeremy** [comment]
Would `translate: 'translateZ(0)'` help in addition to the `willChange` prop?

**Comment by tim-smart** [comment]
> Would `translate: 'translateZ(0)'` help in addition to the `willChange` prop?

No it doesn't help here.

**Comment by tim-smart** [comment]
> Are we sure there's no way to keep the blur but improve the performance? I'm finding it difficult to decide if it's worth losing the blur for the tiny perf improvement.

I think it depends on your gpu. I'm using a Pixel 5 and it is quite painful to use the category selector - really choppy and hard to get to the bottom with lots of categories.

**Comment by youngcw** [comment]
I would think that increasing the darkening effect would probably work to be more obvious.  You could potentially modify the theme files to have darkening that is theme dependent.

**Review by MikesGlitch — APPROVED** [review]
Making it darker helps! :shipit: Will wait for a second review since others have commented.

**Review by youngcw — APPROVED** [review]
Looks good!

**Pull request merged**
